src/dtx/dtx_resync.c

```diff
@@ -518,7 +518,7 @@ dtx_iter_cb(uuid_t co_uuid, vos_iter_entry_t *ent, void *args)
 	} else {
 		/* For pool map refresh. */
 		/* Leader: do nothing. */
-		if ((ent->ie_dtx_flags & DTE_LEADER) && dra->resync_version != dra->discard_version)
+		if ((ent->ie_dtx_flags & DTE_LEADER) && dra->discard_version == 0)
```

Comment:
may refer @Nasf-Fan 's comment of PR9744.
and this PR with quite some same code change as PR9744, won't they conflict?